storybook/src/components/Grid/Grid.docs.mdx

```diff
@@ -70,6 +70,22 @@ Use the `start` prop with 3 values, e.g. `start={{ narrow: 2, medium: 4, wide: 6
 
 <Canvas of={GridStories.StartPosition} />
 
+### Common configurations
```

Comment:
I like that we're describing this, but I'm not sure about the placement. It's tucked away deep in an already quite long page. Isn't this part of the Pages section in Storybook?

Reply:
Yes, now it is!